File: Python/spaceSum.py

s = "   fly me   to   the moon  "

def lengthOfLastWord(s):
    word = s.strip()
    cont = 0
    for x in range(len(word)-1,-1,-1):

        if (word[x] == ' '): break
        cont += 1

    return cont


print(lengthOfLastWord(s))

File: JavaScript/spaceSum.js

s = "   fly me   to   the moon  "

var lengthOfLastWord = function(s) {
    
    word = s.trim()
    cont = 0
    for (let x = word.length - 1; x > -1; x--){
        if (word[x] == ' ') break       
        cont++
    }
    return cont
};

console.log(lengthOfLastWord(s))